assets/src/edit-story/components/canvas/pagemenu/pageMenu.js

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { useCallback } from 'react';
+import { __, sprintf } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Icons } from '../../../../design-system';
+import { useStory, useHistory, useConfig, useCanvas } from '../../../app';
+import { createPage, duplicatePage } from '../../../elements';
+import PageMenuButton from './pageMenuButton';
+import AnimationToggle from './animationToggle';
+
+const DIVIDER_HEIGHT = 16;
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  --pagemenu-icon-space: ${({ isWidePage }) => (isWidePage ? 16 : 10)}px;
+  --pagemenu-count-space: ${({ isWidePage }) => (isWidePage ? 24 : 12)}px;
+`;
+
+const PageCount = styled.div`
+  color: ${({ theme }) => theme.colors.fg.primary};
+  font-family: ${({ theme }) => theme.DEPRECATED_THEME.fonts.body1.family};
+  font-size: ${({ theme }) => theme.DEPRECATED_THEME.fonts.body1.size};
```

Comment:
Should these be moved to the design-system under `typography`, too?

Reply:
I could do [even better actually](https://github.com/google/web-stories-wp/pull/6272/files#diff-e12246607f5678a452584b8ec9cfc1990d830a3db193f87d7bf2b385ed760b0dR114-R120)! Thanks for pushing me when I'm lazy 😉 .